setup.py

```diff
@@ -27,6 +27,7 @@
         zip_safe=False,
         setup_requires=['setuptools>=18.0', 'wheel'],
         install_requires=[
+
```

Comment:
why a space here?

Reply:
This must have been a mistake when merging with main, ill fix that